x-pack/plugin/src/yamlRestTest/resources/rest-api-spec/test/ml/3rd_party_deployment.yml

```diff
@@ -215,16 +215,34 @@ setup:
       catch: /Could not find trained model definition \[distilbert-finetuned-sst\]/
       ml.start_trained_model_deployment:
         model_id: distilbert-finetuned-sst
+
+---
+"Test start deployment fails while model download in progress":
+
+  - do:
+      ml.put_trained_model:
+        model_id: .elser_model_2
```

Comment:
Hmm, I know we've avoided downloading the elser model in tests because it takes a while (which actually helps us in this test). I wonder if this will have any negative affects since the download might continue in the background. I wonder if we could try canceling it during the test cleanup?

Or maybe do a unit test instead 🤷‍♂️

Reply:
yes, thats a good point. I added model deletion and deletion verification to the test.